### DISCUSSION TITLE
What are plans for this project?

### DISCUSSION THREAD

**Discussion by Andre601** [upptime/upptime#618] [General] [answered]
Looking at the discussions here, you can see that Q&A discussions do receive replies from Maintainers, tho other categories, primarily ideas, don't.

That's why I was now wondering how the situation regarding this project as a whole is.
Considering none of the more recent discussions for feature ideas and changes received an answer, it seems like this project entered a "Maintenance only" state, which would be very sad as it's one of if not the only OS-based project using GitHub pages as a hosting solution.

While I don't demand changes to be made ASAP - after all is this a free, open source project while you guys do have a life outside of this - would I still appreciate a general feedback on if there are any changes planned, or if that project really entered a state of only being maintained and nothing more.

Another reason why I believe this project to no longer being actively developed is the fact that even after sending a mail to the e-mail address given on the upptime org account, I so far haven't received any reply whatsoever... Despite it being sent months ago by now.

Again, I don't want to be demanding. I just would like some clarity about this entire project.

**Comment by AnandChowdhary** [comment, marked as answer]
No offense taken, thanks for bringing this up! We planned (albeit the progress has very much slowed down) on releasing a new version of Upptime with a new CLI at its center (https://github.com/upptime/cli) which ideally adds many new features.

Since that's on hold currently, I'm doing my best in fixing bugs and discussing/engaging with the community with my limited free time. When I started Upptime, it was from an idea I had one afternoon; today, I'm the founder and CTO of a company with real employees and millions in funding, so my open source contributions have totally taken a hit :( since I've been putting double-digit hour days.

I would be happy to explore adding new maintainers if you or anyone else is interested in contributing to adding new features! Thanks again!

**Reply by Andre601** [reply]
Thanks for that fast and detailed reply.

I, unfortunately, do not have knowledge in any of the fields needed here (i.e. Svelte for the status page (I recall it being Svelte)), otherwise, would I have been happy to provide contributions already outside of those tiny ones I made (Such as removing that extra text linking to the same issue without any real difference)

**Comment by mchelen** [comment]
@AnandChowdhary Would you be willing to configure this as a crowd funded project such as through [OpenCollective](https://opencollective.com/) or [Github Sponsors](https://github.com/sponsors)? That would allow the community to help pay for other developers to maintain and enhance this project.

**Reply by AnandChowdhary** [reply]
My only hesitation is that when I sponsor a project, I expect them to work hard and I'm a bit afraid of the responsibilities that come with it. I propose that let's first find a developer who would like to maintain, set the budget, and  then set up funding. Pabio (my company) will be happy to cover of it, if not all, but I would like to first have someone approach me to maintain the project. Is  that fair?

**Reply by Andre601** [reply]
Maybe one thing that should be made is a public roadmap (Ideally in this repository using GitHub's project board to make a kanban or what they are called).

I'm sure most people hesitate in contributing since they don't even know what is planned for the future. Not every suggestion may get implemented after all, so a place to see what is planned and up for grabs would already be useful, which brings me to the next point...

...labels. Maybe label the discussions to indicate that stuff is planned and up for grabs for contributors.

Finally, a public post (Pinned issue, blog post, pinned discussion) could also help spreading awareness of future plans.

tl;dr: A better and more open communication regarding future plans.

**Reply by Andre601** [reply]
Coming back to this, I would like to point out, that perhaps the blog could also be utilized here to cover questions (Mainly the "Is this still maintained?" one as I believe many others may ask this too) and perhaps provide a rough roadmap into the direction you want upptime to go in the future.

Perhaps even discuss a bit about the CLI and what it should do in the future?

Because the blog doesn't seem to be used that often I guess and people usually check that amongst other things for if there is noteworthy stuff.
